droidlet/tools/artifact_scripts/tracked_checksums/nlu.txt

```diff
@@ -1 +1 @@
-b15b887d3795140a5fab42a68d7369fcebf28e8c
+2f6a2d5d35f6f3d4ad6730ff831071284d987114
```

Comment:
is this the hash of the model retrained on the updated dataset with the move -> dance change ?

Reply:
> is this the hash of the model retrained on the updated dataset with the move -> dance change ?

Yes.